scss/_type.scss

```diff
@@ -0,0 +1,100 @@
+// Body
```

Comment:
Should these use the mixins that are already defined? @rod?

```.body-1 {
@include body-1;
}```

Reply:
yes